client/src/components/Navbar.js

```diff
@@ -98,9 +98,13 @@ const GetTicketsCTA = () => (
 const NavbarItem = ({to, text, external, pathname}) => {
   let navLinkClass = cn('nav-link', navLink, {active: pathname === `/${to}`});
   const isNew = to === 'sponsors'
+
   return (
     <NavLI key={to}>
-      <LinkDuo className={cn(navLinkClass, isNew? newLink: '')} to={to} external={!!external}>
+      <LinkDuo
+        className={cn(navLinkClass, isNew? newLink: '')}
+        to={to}
+        external={external ? 'true' : undefined}>
```

Comment:
its weird IMO, but something in React throws a Warning when passing a boolean, here I pass a string or an undefined, which is fine, because string is truthy and undefined is falsy

Reply:
Hmmm @yarivShamash I think the reason behind is that we propagate all the props to an `<a>` tag... 
Which is an html element, props become attributes, and a boolean is not valid...

<img width="584" alt="LinkDuo_jsx_—_reversim-summit-2020" src="https://user-images.githubusercontent.com/1729783/68855688-72b68e00-06e7-11ea-8d98-58b8661c0285.png">

Reply:
But removing the ...props form the anchor tag did not help...

Anyway this is the solution - the one that is here in the code review